### PR TITLE
Update past events

### PR DESCRIPTION
**Pull request by MalteRei**
Adding past events and the AI Gaming event to make the website more up-to-date.
@spethso can you please check if the brilliant presentation page is fine
@Dominik-MCC can you please check if the Hamburger technology meetup page is fine

### CODE REVIEW

**Review by spethso — APPROVED** [review]

**Comment by spethso** [comment]
What is the status regarding the AI Gaming event which is in this PR too?

**Comment by MalteRei** [comment]
> What is the status regarding the AI Gaming event which is in this PR too?

Do you mean in general or specific to the website. I also added the page for it in one of the commits. Maybe I will just split the commits into different merge requests to merge what is already reviewed now and wait with the other event pages.

**Comment by spethso** [comment]
> Do you mean in general or specific to the website. I also added the page for it in one of the commits. Maybe I will just split the commits into different merge requests to merge what is already reviewed now and wait with the other event pages.

Specific to the website. :-) I think splitting makes sense until we have approval from Dominik.

**Pull request merged**
